### PR TITLE
docs: update Chinese translation of Deployment

### PR DESCRIPTION
**Pull request by kidonng**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Underlying tools
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**

### CODE REVIEW

File: docs/zh/guide/deployment.md [hunk 1]
```diff
@@ -31,19 +31,17 @@ serve -s dist
...
+
+#### 手动推送更新
  
 1. 在 `vue.config.js` 中设置正确的 `publicPath`。
     
-   如果打算将项目部署到 `https://<USERNAME>.github.io/` 上 , `publicPath` 将默认被设为 `"/"`，你可以忽略这个参数。
+   如果打算将项目部署到 `https://<USERNAME>.github.io/` 上, `publicPath` 将默认被设为 `"/"`，你可以忽略这个参数。
 
-   如果打算将项目部署到 `https://<USERNAME>.github.io/<REPO>/` 上, (换句话说 仓库地址为 `https://github.com/<USERNAME>/<REPO>`), 可将 `publicPath` 设为 `"/<REPO>/"` 。 举个例子， 如果仓库名字为 "my-project"，`vue.config.js` 的内容应如下所示:
+   如果打算将项目部署到 `https://<USERNAME>.github.io/<REPO>/` 上, (换句话说，你的仓库地址为 `https://github.com/<USERNAME>/<REPO>`), 可将 `publicPath` 设为 `"/<REPO>/"` 。 举个例子, 如果仓库名字为 "my-project"，`vue.config.js` 的内容应如下所示:
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
   如果打算将项目部署到 `https://<USERNAME>.github.io/<REPO>/` 上 (即仓库地址为 `https://github.com/<USERNAME>/<REPO>`)，可将 `publicPath` 设为 `"/<REPO>/"`。举个例子，如果仓库名字为“my-project”，那么 `vue.config.js` 的内容应如下所示：
```

File: docs/zh/guide/deployment.md [hunk 2]
```diff
@@ -53,7 +51,7 @@ serve -s dist
     }
     ```
 
-2. 在项目目录下， 用以下的代码创建 `deploy.sh`（可以适当地取消注释）并运行它以进行部署：
+2. 在项目目录下, 创建内容如下的 `deploy.sh`（可以适当地取消注释）并运行它以进行部署：
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
2. 在项目目录下，创建内容如下的 `deploy.sh` (可以适当地取消注释) 并运行它以进行部署：
```

File: docs/zh/guide/deployment.md [hunk 3]
```diff
@@ -83,9 +81,33 @@ serve -s dist
...
     ```
 
-    ::: tip
-    您还可以在 CI 设置中配置上述脚本，以便在每次推送时启用自动部署。
-    :::
+#### 使用 Travis CI 自动更新
+
+1. 仿照上面在 `vue.config.js` 中设置正确的 `publicPath`。
+2. 安装 Travis CLI 客户端：`gem install travis && travis --login`
+3. 生成一个拥有 "repo" 权限的 GitHub [访问令牌](https://help.github.com/cn/articles/creating-a-personal-access-token-for-the-command-line)。
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
3. 生成一个拥有“repo”权限的 GitHub [访问令牌](https://help.github.com/cn/articles/creating-a-personal-access-token-for-the-command-line)。
```

File: docs/zh/guide/deployment.md [hunk 4]
```diff
@@ -83,9 +81,33 @@ serve -s dist
...
+4. 授予 Travis 访问仓库的权限：`travis set GITHUB_TOKEN=xxx` (`xxx` 是第三步中的个人访问令牌)
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
4. 授予 Travis 访问仓库的权限：`travis set GITHUB_TOKEN=xxx` (`xxx` 是第三步中的个人访问令牌)
```

File: docs/zh/guide/deployment.md [hunk 5]
```diff
@@ -130,16 +152,45 @@ module.exports = {
 
 1. 在 Netlify 上，使用以下设置从 GitHub 创建新项目:
 
-    - **构建命令:** `npm run build` or `yarn build`
+    - **构建命令:** `npm run build` 或 `yarn build`
     - **发布目录:** `dist`
 
-2. 点击 deploy 按钮！
+2. 点击 "deploy" 按钮！
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
2. 点击“deploy”按钮！
```

File: docs/zh/guide/deployment.md [hunk 6]
```diff
@@ -130,16 +152,45 @@ module.exports = {
...
 
 也可以查看 [vue-cli-plugin-netlify-lambda](https://github.com/netlify/vue-cli-plugin-netlify-lambda)。
 
+如果使用 Vue Router 的 "history" 模式，你需要在 `/public` 目录下创建一个 `_redirects` 文件：
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
如果使用 Vue Router 的 history 模式，你需要在 `/public` 目录下创建一个 `_redirects` 文件：
```

File: docs/zh/guide/deployment.md [hunk 7]
```diff
@@ -130,16 +152,45 @@ module.exports = {
...
+
+1. 在 Render 上创建一个新的 Web Service，并授予 Render 的 GitHub 应用访问你的 Vue 仓库的权限。
+2. 在创建过程中使用以下设置：
+   - **环境**：`Static Site`
+   - **构建命令**：`npm run build` 或者 `yarn build`
+   - **发布目录**：`dist`
+
+大功告成！构建结束时你的应用便会在你的 Render URL 上线。
+
+如果使用 Vue Router 的 "history" 模式，你需要在站点的 `Redirects/Rewrites` 设置中添加以下改写规则：
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
如果使用 Vue Router 的 history 模式，你需要在站点的 `Redirects/Rewrites` 设置中添加以下改写规则：
```

File: docs/zh/guide/deployment.md [hunk 8]
```diff
@@ -204,55 +255,71 @@ firebase deploy --only hosting
...
 
-    如果想要将项目默认公开部署，部署脚本如下
+```
+now
+```
 
-    ```json
-    "deploy": "npm run build && now --public && now alias"
-    ```
+**此外**，你还可以使用 [GitHub](https://zeit.co/github) 或 [GitLab](https://zeit.co/gitlab) 集成服务。
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
**此外**，你还可以使用他们的 [GitHub](https://zeit.co/github) 或 [GitLab](https://zeit.co/gitlab) 集成服务。
```

File: docs/zh/guide/deployment.md [hunk 9]
```diff
@@ -204,55 +255,71 @@ firebase deploy --only hosting
...
+    {
+      "root": "dist",
+      "clean_urls": true,
+      "routes": {
+        "/**": "index.html"
+      }
+    }
+    ```
+
+3. 将 `static.json` 加入 Git
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
3. 将 `static.json` 加入 git
```

File: docs/zh/guide/deployment.md [hunk 10]
```diff
@@ -310,3 +377,90 @@ npm install --global surge
...
+    docker build . -t my-app
+    # Sending build context to Docker daemon  884.7kB
+    # ...
+    # Successfully built 4b00e5ee82ae
+    # Successfully tagged my-app:latest
+    ```
+
+6. 运行你的 Docker 镜像
+
+    这个例子基于官方 `Nginx` 镜像，因此已经设置了日志重定向并关闭了自我守护进程。它也提供了其他有利于 Nginx 在 Docker 容器中运行的默认设置。更多信息参阅 [Nginx Docker 仓库](https://hub.docker.com/_/nginx).
```
Comment (Justineo) [inline docs/zh/guide/deployment.md]:
```suggestion
    这个例子基于官方 `Nginx` 镜像，因此已经设置了日志重定向并关闭了自我守护进程。它也提供了其他有利于 Nginx 在 Docker 容器中运行的默认设置。更多信息参阅 [Nginx Docker 仓库](https://hub.docker.com/_/nginx)。
```

File: docs/zh/guide/deployment.md (hunk 9 above)
Comment (kidonng) [reply docs/zh/guide/deployment.md]: 个人认为 Git 是专有名词，英语原文没有大写才是不正确的。

Comment (Justineo) [reply docs/zh/guide/deployment.md]: Git 官方 logo 是小写的，但是内文好像是大写的，那就和内文一致吧~

**Pull request merged**